crates/cext/cbindgen.toml

```diff
@@ -25,6 +30,7 @@ prefix_with_name = true
 [export]
 prefix = "Qk"
 renaming_overrides_prefixing = false
+exclude = ["PyObject"]
```

Comment:
This doesn't seem to be working - the generated file still has the prefix.

I'm fine just using `renaming_overrides_prefixing = true` and manually adding `Qk` to the renames for now, while we figure out a nicer pattern.

Probably to do later (because of additional infrastructure), but I think we ought to consider adding a test of the Python-interfacing method.